src/definitions/java/Dockerfile.ubuntu

```diff
@@ -0,0 +1,23 @@
+ARG REGISTRY=docker.io
+ARG IMAGE_TAG=0.0.1-workstation
+ARG ORG=ensono
+ARG JAVA_VERSION=21.0.1
```

Comment:
Although there's plans to upgrade. Stacks is currently on Java 11 being upgraded to 17

Reply:
Downgraded to 17.0.12